services/outputhost/messagecache.go

```diff
@@ -781,6 +786,8 @@ func (msgCache *cgMsgCache) manageMessageDeliveryCache() {
 					msgCache.numAcks = 0
 				}
 			}
+		case <-msgCache.cfgRefreshTicker.C:
+			msgCache.refreshCgConfig(msgCache.maxOutstandingMsgs)
```

Comment:
The creditBatchSize (declared above) is derived out of this maxOutstandingMsgs, so when this gets refreshed, you likely also want to adjust the credit batch size to the store.